src/components/detailedLanding.js

```diff
@@ -20,6 +25,14 @@ const DetailedLanding = ({ userEntered }) => {
   const [focusDone, setFocusDone] = useState(
     localStorage.getItem("focusStatus") === "true" ? true : false
   );
+  useEffect(() => {
+    (async () => {
```

Comment:
suggestion :you can make separate function for that so your code looks more cleaner

Reply:
do you mean custom hook